universal-application-tool-0.0.1/app/services/applicant/question/MultiSelectQuestion.java

```diff
@@ -52,9 +55,8 @@ public boolean hasQuestionErrors() {
 
   @Override
   public boolean hasTypeSpecificErrors() {
-    // Return true if the selected options are not valid options.
-    return getSelectedOptionsValue().isPresent()
-        && !getOptions().containsAll(getSelectedOptionsValue().get());
+    // The question does not recognize selected options not present in the options set
```

Comment:
question: can you help me understand this behavior change? More specifically, what if someone hacks their form submission to include bad data? At what point do we handle this?

Reply:
With the current behavior, if someone did this they could store longs in the array (other types would result in an error on write). On read we filter the list of available options to the ones that match the IDs stored in the DB to get the current selected options. If someone were to write bad IDs into the DB, it would appear in the UI the same as if they answered the question without selecting any options.